modules/server/src/network/index.ts

```diff
@@ -1,10 +1,96 @@
-import { NetworkAggregationInterface } from '@/config/types';
+import { buildClientSchema, getIntrospectionQuery, IntrospectionQuery } from 'graphql';
+import { fetchGql } from './gql';
+import { NetworkAggregationConfig, NetworkAggregationConfigInput } from './types';
 
-export const createSchemaFromNetworkConfig = ({
+const isGqlIntrospectionQuery = (input: any): input is IntrospectionQuery => {
+	return (input as IntrospectionQuery).__schema !== undefined;
```

Comment:
this is the top level field that is defined on gql spec getIntrospectionQuery return type

Reply:
these "is" and "as" always get me mixed up... may I ask for a bit of explanation on the intent here?
e.g. there must be a reason you're using "any" instead of the desired type.

Reply:
https://github.com/overture-stack/arranger/pull/877#issuecomment-2161111471